src/app/containers/Image/index.jsx

```diff
@@ -27,14 +27,15 @@ const getCopyright = copyrightHolder => {
 const shouldLazyLoad = position =>
   !!urlWithPageAnchor() || position[0] > LAZYLOAD_FROM_BLOCK;
 
-const ImageContainer = ({ blocks, position }) => {
+const ImageContainer = ({ blocks, position, shouldPreload }) => {
   if (!blocks) {
     return null;
   }
 
   const rawImageBlock = filterForBlockType(blocks, 'rawImage');
   const altTextBlock = filterForBlockType(blocks, 'altText');
   const captionBlock = filterForBlockType(blocks, 'caption');
+  const ShouldPreLoadLeadImage = position[0] <= 4 && shouldPreload;
```

Comment:
What does `position[0] <= 4` do here? Does it check that the image is located within the first 4 blocks of the page?